ibm/service/appconfiguration/resource_ibm_app_config_snapshot.go

```diff
@@ -0,0 +1,306 @@
+package appconfiguration
+
+import (
+	"fmt"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM/appconfiguration-go-admin-sdk/appconfigurationv1"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"log"
+)
+
+func ResourceIBMIbmAppConfigSnapshot() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceIbmIbmAppConfigSnapshotCreate,
+		Read:     resourceIbmIbmAppConfigSnapshotRead,
+		Update:   resourceIbmIbmAppConfigSnapshotUpdate,
+		Delete:   resourceIbmIbmAppConfigSnapshotDelete,
+		Importer: &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"guid": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "GUID of the App Configuration service. Get it from the service instance credentials section of the dashboard.",
+			},
+			"git_config_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Git config id. Allowed special characters are dot ( . ), hyphen( - ), underscore ( _ ) only",
+			},
+			"git_config_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Git config name. Allowed special characters are dot ( . ), hyphen( - ), underscore ( _ ) only",
+			},
+			"git_url": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Git url which will be used to connect to the github account.",
+			},
+			"git_branch": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Branch name to which you need to write or update the configuration.",
+			},
+			"git_file_path": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Git file path, this is a path where your configuration file will be written.",
+			},
+			"git_token": {
+				Type:        schema.TypeString,
+				Required:    true,
```

Comment:
Can you mark this attribute as sensitive : true since its a kind of secret

Reply:
Addressed